### ISSUE TITLE
View Details of traceable project

### ISSUE THREAD

**Issue by MoeNick** [Giveth/giveth-next#137]
As a user, I want to see the details of a traceable project. 

Details:
The API:

https://feathers.beta.giveth.io/campaigns?verified=true

https://feathers.beta.giveth.io/campaigns/{campaignId}

Campaign link in trace dapp:
https://trace.giveth.io/campaign/{campaignSlug}

UAT Api:

https://feathers.develop.giveth.io/campaigns?verified=true

https://feathers.develop.giveth.io/campaigns/{campaignId}

Campaign link in UAT dapp:

https://develop.giveth.io/campaign/{campaignSlug}

Success Criteria:
Click on a specific traceable project, seee the details: owner, descriptions, traces, leader board, balances, updates and so on.

**Comment by MoeNick** [comment]
based on test scenario, I can't see the detains in uat, such as  owner, descriptions, traces, leader board, balances, updates and so on.

**Comment by mateodaza** [comment]
It's a bit hard for me to get all that info without a design... should I follow your mockup?

**Comment by MoeNick** [comment]
@mateodaza @laurenluz 
This is the concept. Adding traces tab, also show traceable donations beside manual donations (which in this case all of them are traceable) we can bring comments to updates tab, we may bring subscribing concept by adding a button there.
BUT: there is no liking in trace, we can keep it here, and also I think there is no room for leaderboard. and the owners is the same as project manger her.

![Slide1](https://user-images.githubusercontent.com/80677512/128811021-4df0d7af-35f2-4fe8-a7c9-9f73f7e480ba.JPG)
![Slide2](https://user-images.githubusercontent.com/80677512/128811027-3946f9cd-132d-40e4-9eed-f5ed7ef290a1.JPG)
![Slide3](https://user-images.githubusercontent.com/80677512/128811029-9fdd1d10-067a-4842-8c43-427cc8341e11.JPG)
![Slide4](https://user-images.githubusercontent.com/80677512/128811031-7bb7e206-794d-4c18-adba-22b7f81a5057.JPG)
![Slide5](https://user-images.githubusercontent.com/80677512/128811036-75862201-87f0-4e64-b410-cc85e33afa7a.JPG)
![Slide6](https://user-images.githubusercontent.com/80677512/128811041-ac5baca8-8440-4bfa-b46d-11198690962f.JPG)
![Slide7](https://user-images.githubusercontent.com/80677512/128811044-9920458f-5909-498d-8388-023af8a09b6b.JPG)
![Slide8](https://user-images.githubusercontent.com/80677512/128811047-a12f205d-5142-4c71-8b6a-44dccee71891.JPG)
![Slide9](https://user-images.githubusercontent.com/80677512/128811051-7710f703-ea33-4121-9870-a4a02f9c8cbc.JPG)
![Slide10](https://user-images.githubusercontent.com/80677512/128811053-b6fb6fc5-2e89-4444-95df-48e482272fd2.JPG)

**Comment by MoeNick** [comment]
Please consider that this story is **just for viewing** details of campaign. we plan for functionalities later. 
In this story, It is expected we have a view over:
- [ ] donations (list , with counting of total amount and number of donations)
- [ ] traces
- [ ] description

**Comment by laurenluz** [comment]
this is working fine on live!
![image](https://user-images.githubusercontent.com/75490971/130523394-43b7f669-fead-4df8-9413-b6b16581c0d1.png)

**Comment by MoeNick** [comment]
@laurenluz It has a problem fetching traceable or verified project, pls let it be here.

**Comment by MoeNick** [comment]
Mohammad said this:
Mateo call trace.giveth.io in your staging ENV, please use this url in giveth staging, for getting verified projects
https://feathers.develop.giveth.io/campaigns?verified=true

**Comment by MoeNick** [comment]
So lets io staging work with trace develop and for production we gonna use trace.giveth

**Comment by divine-comedian** [comment]
Hey there! I used the project verification tool and now my project shows up twice on giveth.io - one is traceable and the other is not. 

is this in the scope of this issue?

![image](https://user-images.githubusercontent.com/67759413/131544899-108a7d9c-e319-41ba-a5f2-bfa4beca1565.png)


another thing I noticed is that if I check out a traceable project that has no traces yet there is an infinite loading wheel that shows on the page. 

![image](https://user-images.githubusercontent.com/67759413/131545371-cb4aa867-6a24-480d-96f5-190fa784a6b0.png)

**Comment by MoeNick** [comment]
Yeah we have another issue for this, listed under integration epic I think @divine-comedian

**Comment by mateodaza** [comment]
> Yeah we have another issue for this, listed under integration epic I think @divine-comedian

It happens because I'm fetching both traceable and regular projects. Is this going to be solved backend side or should I filter it when I list it?

About the infinite loading wheel, I'll fix it :)

**Comment by MoeNick** [comment]
I think we should remove trace tabs for traceable projects
